### PR TITLE
Slimepeople buffs/nerfs

### PR DESCRIPTION
**Pull request by Incoming5643**
Slimepeople can now split if they contain 200 units of slime jelly, and slimepeople will now slowly generate slime jelly up to 200 units provided they are very well fed.

Split slimepeople are NOT player controlled, but rather the original slime person can swap between the two mobs at will. If one of the slimepeople should die under player control, the player won't be able to swap back to their living body.

Splitting only creates a new body, any items you have on you are not duplicated.

In testing this takes longer to achieve than you might think, even under ideal "FOOD EVERYWHERE" conditions it can take nearly 10 minutes to produce a split slimeperson. The initial slimeperson is also apt to get fat if they try to do this more than once. Best case scenario is for the slimeperson to get an illness that makes them naturally gain weight, but as two of the more isolated jobs on the station I'd be impressed if a xenobiologist and a virologist could work together. 

Slimepeople now take half damage from sources of heat, but double damage from sources of cold. This is more inline with actual slimes, which are IMMUNE to sources of heat (including lasers) but hilariously weak to coldness.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
i like it

**Comment by Iamgoofball** [comment]
>split 5 times
>hide them everywhere
>have them taking 50u nutrient pills 24/7
>have 200 slime clones hidden in maint
>every time you're about to die swap to a clone

how 2 immortality: the race
:+1:

**Comment by Incoming5643** [comment]
Important note on slimepeople splitting, When you make one only you and the new split trade action buttons.

For example say slimeperson A made B, and then B made C. Then later slimeperson B died while the player was in slimeperson C. The player can no longer get back to slimeperson A.

The obvious way around this is to just have one slimeperson doing all the splitting, this is made easier by an obscure mechanic that allows you to eat more the more you eat. Downsides here are obviously if slimeperson prime dies then all your clones save the one you're in become worthless and prime will also become a hidious lardass that's hard to move around.

**Comment by Iamgoofball** [comment]
this is why you play with slimeperson zeta and move up the alphabet when you die

**Comment by Shadowlight213** [comment]
@Incoming5643 There should be an indicator of where the other slimepersons are. Something sorta like the lich message

**Comment by WJohn** [comment]
I can definitely see wizards going slime now, when combined with a gem hardsuit. Having *at least* 50% laser resistance (and then plus whatever the gem hardsuit offers) is gonna come in pretty handy.

**Comment by Bawhoppen** [comment]
>  I'd be impressed if a xenobiologist and a virologist could work together.

Who says you would get slime person from xenobiology as opposed to the FAR easier alternative of using the blood magic machine (aka EXPERIMENTOR)

**Comment by Alek2ander** [comment]
There needs to be a way to invite a ghost to play your clone. And also a way for someone else to recall your ghost to a clone if you managed to die with a living clone.

**Comment by bgobandit** [comment]
how does this work with actual genetics cloning/autocloning

**Comment by Supermichael777** [comment]
i would assume that you have to have the body your mind died in based on how this seems to work.

**Comment by Incoming5643** [comment]
I'll probably end up removing slimepeople from the wizard mirror due to the effective "half damage from projectiles". It's a big bonus for sure, and I can't think of many fun gimmicks a slime wizard would bring to the table otherwise.


@bgobandit If you had backed up the specific body you died in autoprocess and normal cloning should work. The way I'm reading this autoprocess should still work even if you died in a seperate body than the one you initially scanned with because they are genetically identical, but don't quote me on that.

Keep in mind that getting cloned also severs any mind ties you have with other split slimepeople. So the real answer here is "Get off of that critical body before you die and be wary of gibs"

**Comment by AlexCrimson** [comment]
Neat feature, but i feel like its unbalanced for something Scientists can get so easily.

**Comment by Incoming5643** [comment]
It's a late game bonus, something that's practically enshrined in xenobio

**Comment by AlexCrimson** [comment]
Green slimes are not endgame.

**Comment by Incoming5643** [comment]
Taking the time away from xenobio to either eat all the food on the station (and I mean food food, there's mechanics to keep you from eating all the vendor food) or breed up silver slimes simultaneously then wait the 10+ minutes for the nutrition to actually all process (while all the time still eating) makes it significantly more end gamish to me.

Listen I just wanted to make slimepeople more distinct than how they are currently. Their position as the only crew aligned obtainable race that's allowed to be better than a human in terms of gameplay makes them a unique opportunity for me.

**Comment by AlexCrimson** [comment]
Why do you need to eat? Can you not just make Purple Slimes? Those things give like 50 units of Slime Jelly per core. Even so, you can just make some hydroponics trays, get some watermelon seeds, and start your own nutrition farm. Im sure Botany would be willing to trade materials from some Slime Potion, considering its making you functionally immortal if given enough time.

**Comment by Bawhoppen** [comment]
you ignored the part where you can get slime people from the experimentor in literally like 5 minutes

**Comment by Cheridan** [comment]
To those concerned about balance: Seems like a lot more hoops to jump through than just getting a DNA backup at Genetics. Especially if you've got an autocloner setup, which R&D can also make...

**Comment by GunHog** [comment]
I love it :+1:

**Comment by Dorsisdwarf** [comment]
Definitely weaker and harder to get to than an autocloner setup.

Hell, this is harder to get than an autocloner in a sealed room with a teleporter out.

**Comment by Incoming5643** [comment]
if people can get mutant races really easily from the experimentor that's more a balance problem with the experimentor

**Comment by AlexCrimson** [comment]
Autocloner requires materials from Mining. Green Slimes require very simple breeding of Grey>Purple Slimes. You can probably dilute the reagent in water to get multiple uses out of it too.

**Comment by GunHog** [comment]
It is not that big a deal, even if you can easily get them from slimes or risking yourself getting acid-gassed or borged by the Experimentor. Not every mutant race has to be intolerable trash compared to humans.

**Comment by Dorsisdwarf** [comment]
http://static1.squarespace.com/static/52f51a96e4b0ec7646cd474a/5328b57de4b067106916ef7f/5328b931e4b067106916f59f/1395177779500/littlecats1.jpg?format=500w

**Comment by Dorsisdwarf** [comment]
littlecats1

**Comment by KorPhaeron** [comment]
This sounds fun who cares if its slightly unbalanced

**Comment by Incoming5643** [comment]
@Dorsisdwarf conflicts should be resolved

**Comment by Incoming5643** [comment]
@Dorsisdwarf conflicts should be resolved (again)

**Comment by MMMiracles** [comment]
raz pls

File: code/_onclick/hud/action.dm [hunk 1]
```diff
@@ -1,8 +1,3 @@
-#define AB_ITEM 1
-#define AB_SPELL 2
-#define AB_INNATE 3
-#define AB_GENERIC 4
-
```
Comment (Incoming5643) [inline code/_onclick/hud/action.dm]: Another pull had stopped the use of these defines, but they had forgotten to remove them here

**Comment by Incoming5643** [comment]
Resolves conflicts for a third time due to how actions are subtyped now, and thoroughly retested to make sure it all works properly.

ANY DAY NOW GUYS

**Comment by KorPhaeron** [comment]
Never ever

**Comment by Incoming5643** [comment]
FOURTH CONFLICT RESOLUTION
---

**Comment by Iamgoofball** [comment]
@Razharas @hornygranny @bs12-bot @Cheridan @RemieRichards @MrStonedOne @paprka @paprika @god

**Comment by Incoming5643** [comment]
At last, I can unclench

**Comment by KorPhaeron** [comment]
@iamgoofball

How much do your professional mergebegging services cost? I have a PR that could use yout help

**Comment by Iamgoofball** [comment]
@KorPhaeron admin candidate

**Pull request merged**